core/templates/pages/contributor-dashboard-page/services/contribution-opportunities-backend-api.service.ts

```diff
@@ -72,6 +76,10 @@ interface VoiceoverContributionOpportunities {
   more: boolean;
 }
 
+interface FetchReviewableTranslationOpportunitiesResponse {
```

Comment:
Should there be a verb at the start of the interface? It seems quite weird. Maybe "Fetched"?

Reply:
Sure. Done.